modules/system/assets/ui/js/waiaria.js

```diff
@@ -85,7 +87,7 @@ $(function() {
      */
     $('body').on('change', '.switch-field input[role="checkbox"]', function () {
```

Comment:
@ayumihamsaki Will `event` need to be added to the function parameters here and on line 113 too?
```suggestion
    $('body').on('change', '.switch-field input[role="checkbox"]', function (event) {
```

Reply:
Done, thanks 👍